### ISSUE TITLE
Empty 'title' in function add_uri_to_queue() caused an AvTransport error

### ISSUE THREAD

**Issue by pfischi** [SoCo/SoCo#448]
An error is thrown by the utils.py function 'really_unicode()' if an uri without metadata (especially with empty title) is added to the queue by 'add_uri_to_queue()'. I my case, i've added a local mp3-file tothe queue as follows:

```soco.add_uri_to_queue('x-rincon-mp3radio://192.168.178.22:8802/ZGVfX0hhbGxvIEhhbGxvIER1.mp3')```

The file is played correctly but this track caused errors triggered by AVTransport events.

```Traceback (most recent call last):
  File "/usr/lib/python3.4/socketserver.py", line 617, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.4/socketserver.py", line 344, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.4/socketserver.py", line 673, in __init__
    self.handle()
  File "/usr/lib/python3.4/http/server.py", line 398, in handle
    self.handle_one_request()
  File "/usr/lib/python3.4/http/server.py", line 386, in handle_one_request
    method()
  File "/home/sb/projects/shSonos/server.sonos/soco/events.py", line 229, in do_NOTIFY
    variables = parse_event_xml(content)
  File "/home/sb/projects/shSonos/server.sonos/soco/events.py", line 130, in parse_event_xml
    value = from_didl_string(value)[0]
  File "/home/sb/projects/shSonos/server.sonos/soco/data_structures.py", line 99, in from_didl_string
    items.append(cls.from_element(elt))
  File "/home/sb/projects/shSonos/server.sonos/soco/data_structures.py", line 513, in from_element
    title = really_unicode(title_elt.text)
  File "/home/sb/projects/shSonos/server.sonos/soco/utils.py", line 43, in really_unicode
    raise ValueError('%s is not a string at all.' % in_string)
ValueError: None is not a string at all.
```

If i change the line in "add_uri_to_queue()" from 

```item = DidlObject(resources=res, title='', parent_id='', item_id='')```
to
```item = DidlObject(resources=res, title='here a title', parent_id='', item_id='')```

everything seems to be fine.

Stefan

**Comment by KennethNielsen** [comment]
Thanks for reporting this. @ghcs27 made a PR to fix this (#467). If you have the option to test that branch that would be great.

**Comment by DPH** [comment]
@pfischi - not heard back for some time so closing this issue. 
If still a problem please re-open issue. Cheers David
